From b3ff26fa182b34b6faa1317f18a9dac4c7963334 Mon Sep 17 00:00:00 2001
From: Matt Weber <matthew.weber@rockwellcollins.com>
Date: Mon, 2 Oct 2017 09:57:15 -0500
Subject: [PATCH] nfs-utils: add missing include of stdint.h

Glibc bump to 2.26 exposed this missing header when building
with the following combination using an i386 internal toolchain.
gcc5.4.0
bin2.28.1
linux4.1.43

Upstream: https://bugzilla.linux-nfs.org/show_bug.cgi?id=312

Signed-off-by: Matthew Weber <matthew.weber@rockwellcollins.com>
---
 support/nsm/rpc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/support/nsm/rpc.c b/support/nsm/rpc.c
index 4e5f40e..d91c6ea 100644
--- a/support/nsm/rpc.c
+++ b/support/nsm/rpc.c
@@ -40,6 +40,7 @@
 
 #include <time.h>
 #include <stdbool.h>
+#include <stdint.h>
 #include <string.h>
 #include <unistd.h>
 #include <fcntl.h>
-- 
1.9.1

